test/hlint.hs

```diff
@@ -22,6 +22,8 @@ main = do
         , "--ignore=Use unless"
         , "--ignore=Reduce duplication"
         , "--cpp-define=USE_TEMPLATE_HASKELL"
+        , "--ignore=Use >>"
+        , "--ignore=Fuse foldr/map"
```

Comment:
In newer version of hlint it now suggests
```
linting file src/Reflex/Dynamic/TH.hs... /home/luigy/code/reflex-platform-reunification/reflex/src/Reflex/Dynamic/TH.hs:50:13: Suggestion: Fuse foldr/map
Found:
  foldr (\ a b -> ConE 'FHCons `AppE` a `AppE` b) (ConE 'FHNil) $
    map snd exprs
Perhaps:
  foldr ((\ a b -> ConE 'FHCons `AppE` a `AppE` b) . snd)
    (ConE 'FHNil)
    exprs

/home/luigy/code/reflex-platform-reunification/reflex/src/Reflex/Dynamic/TH.hs:51:15: Suggestion: Fuse foldr/map
Found:
  foldr (\ a b -> ConP 'HCons [VarP a, b]) (ConP 'HNil []) $
    map fst exprs
Perhaps:
  foldr ((\ a b -> ConP 'HCons [VarP a, b]) . fst) (ConP 'HNil [])
    exprs

2 hints
```

Reply:
I'm curious why we wouldn't want these lints to just be applied?